include/picolibrary/testing/interactive/gpio.h

```diff
@@ -30,6 +30,58 @@
  */
 namespace picolibrary::Testing::Interactive::GPIO {
 
+/**
+ * \brief GPIO input pin state interactive test helper.
+ *
+ * \tparam Output_Stream The type of asynchronous serial output stream to use to output
+ *         information to the user.
+ * \tparam Transmitter The type of asynchronous serial transmitter to use to transmit
+ *         information to the user.
+ * \tparam Input_Pin The type of input pin to toggle.
+ * \tparam Delayer A nullary functor called to introduce a delay each time the pin's state
+ *         is read
+ *
+ * \param[in] transmitter The asynchronous serial transmitter to use to transmit
+ *            information to the user.
+ * \param[in] pin The pin to toggle.
```

Comment:
This interactive test program does not toggle a pin, it gets the state of a pin. Please update this to something along the lines of:
```suggestion
 * \param[in] pin The pin to get the state of.
```